resourcemanager/commonids/key_vault_test.go

```diff
@@ -6,9 +9,6 @@ import (
 	"github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids"
 )
 
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See NOTICE.txt in the project root for license information.
-
```

Comment:
Are these supposed to be removed?

Reply:
yep, they were supposed to be switched in #154 but I forgot to push the commit 🙃